frontend/components/organisms/documents/DocumentUploadForm.vue

```diff
@@ -107,18 +108,22 @@ export default {
     },
     create() {
       if (this.validate()) {
-        this.uploadDocument({
-          projectId: this.$route.params.id,
-          format: this.selectedFormat.type,
-          file: this.file
+        const promises = []
+        this.file.forEach((item) => {
+          promises.push(
+            this.uploadDocument({
+              projectId: this.$route.params.id,
+              format: this.selectedFormat.type,
+              file: item
+            })
+              .catch(() => {
+                this.showError = true
+              }))
+        })
+        Promise.all(promises).then(() => {
+          this.reset()
+          this.cancel()
```

Comment:
Dialog closes even if any request was unsuccessful and an error wants to be shown.